src/Microsoft.Azure.CosmosEventSourcing/Stores/IEventStore.cs

```diff
@@ -23,6 +23,37 @@ public interface IEventStore<TEventItem> where TEventItem : EventItem
         IEnumerable<TEventItem> items,
         CancellationToken cancellationToken = default);
 
+    /// <summary>
+    /// Persists a set of <see cref="EventItem"/> items.
+    /// </summary>
+    /// <param name="aggregateRoot">The aggregate containing the events to persist.</param>
+    /// <param name="cancellationToken">A token that can be used to cancel this async request.</param>
```

Comment:
Again some remarks about using the attribute here.

Reply:
Added some remarks